src/components/profile/edit/EditPersonalInfo.vue

```diff
@@ -503,13 +503,13 @@ export default {
     Fluent,
   },
   computed: {
-    loggedInUser() {
-      return this.$store.state.user;
+    loggedInUsername() {
+      return this.$store.state.scope.username;
```

Comment:
If we are just using this state.scope situation, we should really be using vuex mapGetters